app/src/main/java/org/jellyfin/mobile/ApplicationModule.kt

```diff
@@ -55,4 +54,7 @@ val applicationModule = module {
     fragment { ConnectFragment() }
     fragment { WebViewFragment() }
     fragment { PlayerFragment() }
+
+    // Media
+    factory { LibraryBrowser(get(), get(), get(), get(), get(), get(), get(), get(), get(), get())  }
```

Comment:
^ Might want to change this one to use `inject()` calls instead of all the constructor parameters